includes/payment-methods/Configs/Definitions/AlipayDefinition.php

```diff
@@ -0,0 +1,308 @@
+<?php
+/**
+ * Alipay Payment Method Definition
+ *
+ * @package WCPay\PaymentMethods\Configs\Definitions
+ */
+
+namespace WCPay\PaymentMethods\Configs\Definitions;
+
+use WCPay\PaymentMethods\Configs\Interfaces\PaymentMethodDefinitionInterface;
+use WCPay\PaymentMethods\Configs\Constants\PaymentMethodCapability;
+use WCPay\Constants\Country_Code;
+use WCPay\Constants\Currency_Code;
+use WCPay\PaymentMethods\Configs\Utils\PaymentMethodUtils;
+
+/**
+ * Class implementing the Alipay payment method definition.
+ */
+class AlipayDefinition implements PaymentMethodDefinitionInterface {
+
+	/**
+	 * Get the internal ID for the payment method
+	 *
+	 * @return string
+	 */
+	public static function get_id(): string {
+		return 'alipay';
+	}
+
+	/**
+	 * Get the keywords for the payment method
+	 *
+	 * @return string[]
+	 */
+	public static function get_keywords(): array {
+		return [ 'alipay' ];
+	}
+
+	/**
+	 * Get the Stripe payment method ID
+	 *
+	 * @return string
+	 */
+	public static function get_stripe_id(): string {
+		return PaymentMethodUtils::get_stripe_id( self::get_id() );
+	}
+
+	/**
+	 * Get the customer-facing title of the payment method
+	 *
+	 * @param string|null $account_country Optional. The merchant's account country.
+	 *
+	 * @return string
+	 */
+	public static function get_title( ?string $account_country = null ): string {
+		return __( 'Alipay', 'woocommerce-payments' );
+	}
+
+	/**
+	 * Get the customer-facing description of the payment method
+	 *
+	 * @return string
+	 */
+	public static function get_description(): string {
+		return __( 'Alipay is a popular wallet in China, operated by Ant Financial Services Group, a financial services provider affiliated with Alibaba.', 'woocommerce-payments' );
+	}
+
+	/**
+	 * Is the payment method a BNPL (Buy Now Pay Later) payment method?
+	 *
+	 * @return boolean
+	 */
+	public static function is_bnpl(): bool {
+		return PaymentMethodUtils::is_bnpl( self::get_capabilities() );
+	}
+
+	/**
+	 * Is the payment method a reusable payment method?
+	 *
+	 * @return boolean
+	 */
+	public static function is_reusable(): bool {
+		return false;
+	}
```

Comment:
I think we should use the util here as well.

```suggestion
	public static function is_reusable(): bool {
		return PaymentMethodUtils::is_reusable( self::get_capabilities() );
	}
```

Reply:
Ah, ok! I was unsure if I could have just returned `false` here, since there's no logic that needs to be executed - the "reusable" attribute is "static" (it doesn't need to be computed)/it's a capability of the payment method itself (some allow tokenization for subscriptions, others don't).